src/components/Body.jsx

```diff
@@ -0,0 +1,41 @@
+import {useEffect, useState} from 'react';
+import getArticles from '../utils/getArticles';
+
+const Body = () => {
+
+    const [filter, setFilter] = useState('none');
```

Comment:
If you want to filter the articles you get this is something that could be done by your API utils if you have your queries set up on the backend :)

Reply:
Thanks Duncan!